tests/throughput-test-harness.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+
+import re
+import sys
+import time
+import logging
+import argparse
+import subprocess
+
+from dockerfile_parse import DockerfileParser
+
+DEFAULT_BRANCH_PREFIX = "branch"
+DEFAULT_BRANCH_COUNT = 100
+DEFAULT_STAGGER_NUMBER = 10
+DEFAULT_STAGGER_WAIT = 60
+DEFAULT_KOJI_BIN = "brew-test"
+DEFAULT_KOJI_TARGET = "extras-rhel-7.2-candidate"
+DEFAULT_GIT_REMOTE = "origin"
+
+def run(*args):
+    logging.info("running: %s", " ".join(args))
+
+    p = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (out, err) = p.communicate()
+    if out:
+        logging.info("stdout:\n%s", out.rstrip())
+    if err:
+        logging.info("stderr:\n%s", err.rstrip())
+    if p.returncode != 0:
+        raise RuntimeError("Subprocess failed w/ return code {0}".format(p.returncode))
```

Comment:
I'd rather fail by default, especially for the git commands, to avoid working on a wrong branch if a checkout fails or pushing something that is not in some expected state. It probably makes sense for starting builds, though - I'll update it.